tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -1,4 +1,5 @@
 import contextlib
+from mock import MagicMock
```

Comment:
Please use `unittest.mock` instead of `mock` (c.f., https://github.com/optuna/optuna/pull/1121)

Reply:
Thank you. CI is succeeded at 8fb0c69.